content/guides/deploying-your-app.mdx

```diff
@@ -0,0 +1,384 @@
+
+3.  Follow the on-screen instructions from the CLI. When prompted for the 'Directory to deploy,' specify `dist` — this is where Solid stores the built project files.
+
+After completing the process, your project will be deployed on Netlify and can be accessed via the provided URL.
+
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=GFp0g1qxMv214z7vjKVYjw)
+
+## Railway
+
+[Railway](https://railway.app/) is a well-known platform for deploying a variety of web and cloud-based projects. 
+For an in-depth look at the features offered by Railway, as well as detailed deployment guidelines, you can consult the [Railway documentation](https://docs.railway.app/).
+
+### Adjust the Start command
+
+To begin, you need to update the start command in your `package.json` file to make it compatible with Railway. 
+Change the start command to `npx http-server ./dist` instead of using `vite`. 
+This adjustment means you will need to build the app to generate the `dist` folder.
+
+For local development, continue using the original `dev` command.
+Reserve the modified start command specifically for Railway deployments. 
+Below is an example of how your `package.json` may be configured:
+
+```jsonl
+"scripts": {
+  "start": "npx http-server ./dist",
+  "dev": "vite",
+  "build": "vite build",
+  "serve": "vite preview",
+  "predeploy": "npm run build",
+  "deploy": "gh-pages -d build"
+},
+```
+
+### Using the Railway Web Interface
+
+1. Visit Railway's homepage and click "Start a New Project."
+You will be redirected to connect with GitHub. 
+Log in or create an account using your GitHub credentials and authorize Railway to access your account.
+
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=Ow2cGMhUtdPVAMoioe4fPA)
+
+2. After authorization, choose the repository that has your Solid project. 
+During this step, you can also add any required environment variables.
+
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=Ixs7AXXAc1xBYz-ecxl0Bg)
+
+3. Once your project is configured, click "Deploy Now." 
+After a successful deployment, a confirmation screen will appear.
+
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=nFgehyw9gH9l7ezv5W4EvQ)
+
+4. Railway does not automatically assign a domain to your project.
+To do this, go to the settings and manually generate a domain for your deployed project.
+
+[](https://app.eraser.io/workspace/w9y9PNVjwSqDCEPNTEoe?elements=2y6TnvBnztCwN8ecamrffQ)
+
+Once a domain has been generated, your Solid project should be live.
+
+### Using the Railway CLI
+
+1. Using your preferred package manager and install the Railway CLI:
+
+```bash
+npm i -g @railway/cli
+# or
+pnpm i -g @railway/cli
+# or
+yarn global add @railway/cli
+```
+
+2. Open your terminal and run the following command to log in:
+
+```bash
+railway login
+```
+
+3. You have the option to link your local Solid project to an existing Railway project using railway link. 
```

Comment:
> railway link

Is this a proper noun? Should it be capitalized?

Reply:
it's a cli command so no capitalization